src/django/api/views/submission.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.viewsets import GenericViewSet
+from rest_framework.mixins import ListModelMixin
+from rest_framework.permissions import IsAuthenticated
+
+from ..models.submission import Submission
+from ..serializers.submission import SubmissionSerializer
+from ..filters.submission import (
+    UserSubmissionFilterBackend,
+    RequestSubmissionFilterBackend,
+)
+
+
+class SubmissionsViewSet(ListModelMixin, GenericViewSet):
+    queryset = Submission.objects.all()
+    serializer_class = SubmissionSerializer
+    permission_classes = [IsAuthenticated]
+    filter_backends = [
+        UserSubmissionFilterBackend,
+        RequestSubmissionFilterBackend,
```

Comment:
Consider using simpler names that communicate what this does, rather than what it is:

```suggestion
        ByRole,
        ByUtility,
```